documentation/pole-workspace-guide.adoc

```diff
@@ -0,0 +1,419 @@
+
+.Different types of crimes
+[source,cypher]
+----
+MATCH (c:Crime)
+RETURN c.type AS crime_type, count (c) AS total
+ORDER BY count(c) DESC
+----
+
+If you recall the graph model, a crime can involve a person, a vehicle or an object.
+
+The following query lets you see which crime(s) involved an object:
+
+.Crimes involving an object
+[source,cypher]
+----
+MATCH (o:Object)-[:INVOLVED_IN]->(c:Crime)
+RETURN c.type AS crime_type, count(c) AS total
+ORDER BY count(c) DESC
+----
+
+[NOTE]
+.Challenge
+====
+Can you rewrite the query to show the crimes that involved a person instead of an object?
+
+[source,cypher]
+----
+MATCH (o:Object)-[:INVOLVED_IN]->(c:Crime)
+RETURN c.type AS crime_type, count(c) AS total
+ORDER BY count(c) DESC
+----
+
+Hint: If you don't remember the data model, you can always run `CALL db.schema.visualization()` to see it again.
+====
+
+[%collapsible]
+.Reveal the solution
+====
+[source,cypher]
+----
+MATCH (p:Person)-[:PARTY_TO]->(c:Crime)
+RETURN c.type AS crime_type, count(c) AS total
+ORDER by count(c) DESC
+----
+====
+
+In the next section you will refactor properties and look at locations in the graph.
+
+== Locations
+
+The `point` data type allows you to use location based functions in Cypher.
+Data Importer doesn't support natively creating `point` data types.
+In order to work with locations in the POLE dataset, you need to create a `point` property on the `Location` nodes.
+Currently the `Location` nodes have a `latitude` and `longitude` property and you can use these to create a `point` property.
+
+.Refactor `Location` nodes
+[source,cypher]
+----
+MATCH (l:Location)
+SET l.position = point({latitude: l.latitude, longitude: l.longitude})
+----
+
+Which locations have the highest crime rate?
+The dataset contains a lot of locations, so it is sensible to put a limit on the number of locations returned.
+
+.Locations with the highest crime rate
+[source,cypher]
+----
+MATCH (l:Location)<-[:OCCURRED_AT]-(:Crime)
+RETURN l.address AS locale, l.postcode AS postcode, count(l) AS total
+ORDER BY count(l) DESC
+LIMIT 20
+----
+
+This query matches locations with crimes returns the `address` and `postcode` properties of the `Location`nodes` and counts all non-null occurences crimes that occurred at that location and orders the results in descending order.
+The `LIMIT` clause limits the number of results returned.
+
+In the next section, you'll use the newly refactored `point` property and Cypher spatial functions.
+
+== Distance
+
+Instead of multiple addresses, you can find out how much and what types of crime were committed in the vicinity of one address.
+You can pick any address as your starting point, but for this query you will use an address that may sound familiar.
+
+.Crimes committed in the vicinity of Coronation Street
+[source,cypher]
+----
+MATCH (l:Location {address: '1 Coronation Street'})
+WITH point(l) AS corrie
```

Comment:
Do you need to use `point(l)`? As you have already created a point datatype could you use 

```
WITH l.position AS corrie
```

Reply:
No, ofc. I've amended.